pkg/apiclients/testapi/oapi.gen.go

```diff
@@ -1,6 +1,6 @@
 // Package testapi provides primitives to interact with the openapi HTTP API.
 //
-// Code generated by github.com/oapi-codegen/oapi-codegen/v2 version v2.4.1 DO NOT EDIT.
+// Code generated by github.com/oapi-codegen/oapi-codegen/v2 version (devel) DO NOT EDIT.
```

Comment:
Question: Is it intentional that the oapi-codegen version is `devel`? not sure if this has an effect though